libmisc/console.c

```diff
@@ -70,7 +70,8 @@ static bool is_listed (const char *cfgin, const char *tty, bool def)
 
 	if (*cons != '/') {
 		char *pbuf;
-		strcpy (buf, cons);
+		strncpy (buf, cons, sizeof (buf));
```

Comment:
Agreed on principle, although I don't believe overflow can happen in practice just bc we always pass in CONSOLE.
So I definately want this, thanks.

Reply:
The overflow can happen. It is not a security issue because the login.defs file is controlled by sysadmin, but still possible crashes are uggly.